types/service/database_service_configuration.go

```diff
@@ -1,15 +1,27 @@
 package service
 
-import "github.com/borderzero/border0-go/types/common"
+import (
+	"errors"
+	"fmt"
+
+	"github.com/borderzero/border0-go/lib/types/slice"
+	"github.com/borderzero/border0-go/types/common"
+)
 
 // Database service types supported by Border0. Choose `standard` for self-managed databases.
-// Use `aws_rds` for AWS RDS databases, and select `google_cloudsql` for Google Cloud SQL databases.
+// Use `aws_rds` for AWS RDS databases, and select `gcp_cloudsql` for Google Cloud SQL databases.
 const (
-	DatabaseServiceTypeStandard = "standard"        // standard MySQL or PostgreSQL, supports TLS and password auth
-	DatabaseServiceTypeRds      = "aws_rds"         // AWS RDS database, supports IAM and password auth
-	DatabaseServiceTypeCloudSql = "google_cloudsql" // Google Cloud SQL database, supports IAM, TLS and password auth
+	DatabaseServiceTypeStandard    = "standard"     // standard MySQL or PostgreSQL, supports TLS and password auth
+	DatabaseServiceTypeAwsRds      = "aws_rds"      // AWS RDS database, supports IAM and password auth
+	DatabaseServiceTypeGcpCloudSql = "gcp_cloudsql" // Google Cloud SQL database, supports IAM, TLS and password auth
 )
 
+var ValidDatabaseServiceTypes = []string{
```

Comment:
the other validators just use a default clause in their switch statements to handle invalid types